CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * [#5561](https://github.com/bbatsov/rubocop/issues/5561): Fix `Lint/ShadowedArgument` false positive with shorthand assignments. ([@akhramov][])
 
+### Changes
+
+* [#5734](https://github.com/bbatsov/rubocop/pull/5734): Change `Naming/UncommunicativeMethodParamName` add `by` to allowed names in default config. ([@AlexWayfer][])
```

Comment:
I think you meant to write `to add` or something like this. Reads a bit odd right now.

Reply:
I just copied from #5626, sorry. Changed.